customer/src/main/java/no/unit/nva/customer/get/GetControlledVocabularyHandler.java

```diff
@@ -40,8 +61,19 @@ protected Integer getSuccessStatusCode(Void input, VocabularySettingsList output
         return HttpURLConnection.HTTP_OK;
     }
 
-    @Override
-    protected List<MediaType> listSupportedMediaTypes() {
-        return SUPPORTED_MEDIA_TYPES;
+    private static CustomerService defaultCustomerService() {
+        AmazonDynamoDB client = defaultDynamoDbClient();
+
+        return new DynamoDBCustomerService(client, ObjectMapperConfig.objectMapper, ENVIRONMENT);
+    }
+
+    private static AmazonDynamoDB defaultDynamoDbClient() {
```

Comment:
I think default amazon clients should be in commons. I want to test how much of a change this configuration actually does.